advanced/Scripts/piholeDebug.sh

```diff
@@ -907,7 +907,7 @@ parse_file() {
         #shellcheck disable=SC2016
         IFS=$'\r\n' command eval 'file_info=( $(cat "${filename}") )'
     else
-        read -a file_info <<< $filename
+        read -a -r file_info <<< $filename
```

Comment:
-r is just a(n) argument

Reply:
(I know i'm talking to a bot ok?)

Reply:
@mrmajik45 stickler is picking up on $filename not being quoted, not on your addition. 
You might like to change the line to 
```        read -a -r file_info <<< "$filename"```
then it will make it through.